src/liger_kernel/transformers/model/gemma.py

```diff
@@ -97,7 +97,7 @@ def lce_forward(
     loss = None
     logits = None
 
-    if self.training:
+    if self.training and labels is not None:
```

Comment:
```suggestion
    if self.training and (labels is not None):
```

truly the **last** suggestion!